lib/web_ui/lib/src/engine/html/shaders/shader.dart

```diff
@@ -56,8 +56,8 @@ class GradientSweep extends EngineGradient {
     double centerX = (center.dx - shaderBounds.left) / (shaderBounds.width);
     double centerY = (center.dy - shaderBounds.top) / (shaderBounds.height);
     gl.setUniform2f(tileOffset,
-        shaderBounds.left + 2 * (shaderBounds.width * (centerX - 0.5)),
-        -shaderBounds.top - 2 * (shaderBounds.height * (centerY - 0.5)));
+        2 * (shaderBounds.width * (centerX - 0.5)),
+        2 * (shaderBounds.height * (centerY - 0.5)));
```

Comment:
nit: a trailing comma could help dartfmt do a better job

Reply:
3 param function. I try to do it for lists and widget trees